CONTRIBUTING.md

```diff
@@ -24,11 +24,11 @@ As for now to get access to this chat please write email to [Sergii Diachenko](m
 
 ## How Can I Contribute? ##
 ### Reporting Bugs ###
-Log issue as __bug__ in the [issue tracker](https://bitbucket.org/VolleyManagement/volleymanagement/issues?status=new&status=open).
+Log issue as __bug__ in the [issue tracker](https://github.com/VolleyManagement/volley-management/issues?q=is%3Aopen+is%3Aissue).
 
 Before creating bug reports, please check existing open issues as you might find out that you don't need to create one. When you are creating a bug report, please include as many details as possible.
 ### Suggesting Improvements ###
-Log issue as __enhcancement__ or __proposal__ in the [issue tracker](https://bitbucket.org/VolleyManagement/volleymanagement/issues?status=new&status=open).
+Log issue as __enhcancement__ or __proposal__ in the [issue tracker](https://github.com/VolleyManagement/volley-management/issues?q=is%3Aopen+is%3Aissue).
```

Comment:
Do we need a filter here?

Reply:
I used just to be sure that we will get open and newest issues.
But it seems work such a way by default.
I can remove.